package.json

```diff
@@ -18,14 +17,15 @@
   "license": "MIT",
   "devDependencies": {
     "coffee-script": "^1.9.1",
-    "docha": "thompsnm/docha#36f8d21",
+    "cucumber": "^0.5.0",
+    "docha": "^0.1.2",
     "grunt": "^0.4.5",
     "grunt-cli": "^0.1.13",
     "grunt-coffeelint": "^0.0.13",
     "grunt-contrib-coffee": "^0.13.0",
     "grunt-contrib-connect": "^0.9.0",
     "grunt-contrib-requirejs": "^0.4.4",
```

Comment:
So they only need to be loaded into the browser? And I guess they're not exported in a way that works with out requirejs?